rhel9/README.md

```diff
@@ -8,7 +8,14 @@ The Packer template in this directory creates a RHEL 9 AMD64 image for use with
 
 * A machine running Ubuntu 22.04+ with the ability to run KVM virtual machines.
 * qemu-utils, libnbd-bin, nbdkit and fuse2fs
-* [Packer](https://www.packer.io/intro/getting-started/install.html), v1.8.0 or newer
+* packer - from Hashicorp repository, v1.9.0 or newer
```

Comment:
keep the reference to Packer install page. It's easier to keep docs up to date if we rely on external resources.